tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+    "compilerOptions": {
+        "target": "ES5",
+        "module": "commonjs",
+        "resolveJsonModule": true,
+        "esModuleInterop": true,
+        "sourceMap": true,
+        "outDir": "build/dist",
+        "noImplicitAny": false,
+        "strict": false
```

Comment:
Add `"declaration": true"`.

Optionally we can add `"declarationMap": true`.

Reply:
I don't think we can add this now to "Generates corresponding d.ts files."

When I add it `tsc` just crashes with the following error:
```
~/airtable/airtable.js% node_modules/typescript/bin/tsc 
/home/rebecca/airtable/airtable.js/node_modules/typescript/lib/tsc.js:82902
                throw e;
                ^

TypeError: Cannot read property 'parent' of undefined
    at transformTopLevelDeclaration (/home/rebecca/airtable/airtable.js/node_modules/typescript/lib/tsc.js:76323:55)
    at visitDeclarationStatements (/home/rebecca/airtable/airtable.js/node_modules/typescript/lib/tsc.js:76270:26)
    at Object.visitNodes (/home/rebecca/airtable/airtable.js/node_modules/typescript/lib/tsc.js:63024:48)
    at transformRoot (/home/rebecca/airtable/airtable.js/node_modules/typescript/lib/tsc.js:75725:37)
    at transformation (/home/rebecca/airtable/airtable.js/node_modules/typescript/lib/tsc.js:76815:24)
    at transformRoot (/home/rebecca/airtable/airtable.js/node_modules/typescript/lib/tsc.js:76833:82)
    at Object.map (/home/rebecca/airtable/airtable.js/node_modules/typescript/lib/tsc.js:335:29)
    at Object.transformNodes (/home/rebecca/airtable/airtable.js/node_modules/typescript/lib/tsc.js:76820:30)
    at emitDeclarationFileOrBundle (/home/rebecca/airtable/airtable.js/node_modules/typescript/lib/tsc.js:77362:43)
    at emitSourceFileOrBundle (/home/rebecca/airtable/airtable.js/node_modules/typescript/lib/tsc.js:77278:13)
```

I'm not confident the source of the error, but I think it might make more sense to try and add this config when the files have valid types.